Fix a build error

bvanassche opened the pull request:
Fix the following build error:

sg_z_act_query.c:529:19: error: too many arguments to function call, expected 0,
      have 1
            usage(1);
            ~~~~~ ^

Fixes: 20315aa4fae1 ("sg_z_act_query: new utility")
Signed-off-by: Bart Van Assche <bvanassche@acm.org>

doug-gilbert commented:
Wow, gcc 10.3.0 with -Wall -W did _not_ report this parameter mismatch. clang does report it. So when I built on FreeBSD which uses clang by default, then the mismatch was reported. Anyway I fixed it upstream and it is now mirrored here (rev 924).

bvanassche committed:
Enable continuous integration

Build sg3_utils upon every push and pull request.

Signed-off-by: Bart Van Assche <bvanassche@acm.org>

bvanassche committed:
Fix a MinGW link error

Fix the following MinGW link error:

libtool: link: x86_64-w64-mingw32-gcc -Wall -W -Wall -Wextra -Werror -Wno-sign-compare -Wno-unused-function -Wno-unused-parameter -D__USE_MINGW_ANSI_STDIO -D__USE_MINGW_ANSI_STDIO -o .libs/sg_timestamp.exe sg_timestamp.o  ../lib/.libs/libsgutils2.dll.a -L/usr/local/lib
/usr/bin/x86_64-w64-mingw32-ld: sg_turs.o:sg_turs.c:(.text+0x73d): undefined reference to `nanosleep'

Signed-off-by: Bart Van Assche <bvanassche@acm.org>

bvanassche committed:
sg_dd: Remove a set-but-not-used variable

Signed-off-by: Bart Van Assche <bvanassche@acm.org>
